WebRoot/messages/ZmMsg.properties

```diff
@@ -4222,6 +4222,8 @@ COLUMN_WIDTH_TYPE_CN = 20
 
 # filter rules and zimlet list
 COLUMN_WIDTH_ACTIVE = 50
+COLUMN_WIDTH_ZIMLET_ACTIVE = 65
```

Comment:
I was not editing existing property as they are used in some other views, also.
So, created new one.
After discussion, it's fine to alter existing property value.
Done. https://github.com/Zimbra/zm-web-client/pull/98/commits/6a6a14dd2f05f961c3d5d8d962c046d40c19d080